lib/timeline/Timeline.js

```diff
@@ -109,6 +104,11 @@ function Timeline (container, items, groups, options) {
   // current time bar
   this.currentTime = new CurrentTime(this.body);
   this.components.push(this.currentTime);
+  
+  // apply options
+  if (options) {
```

Comment:
Here we have the braces. Is this some Javascript style convention to omit them if it's a return? I'm not sure.